about_edit.html

```diff
@@ -1 +1,179 @@
+          <ul class="nav nav-tabs">
+            <li role="presentation" class="col-md-offset-2">
+              <a href="timeline.html">Timeline</a>
+            </li>
+            <li role="presentation" class="active">
+              <a href="about.html">About</a>
+            </li>
+            <li role="presentation">
+              <a href="photos.html">Photos (123)</a>
+            </li>
+            <li role="presentation">
+              <a href="friends.html">Friends (542)</a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </header>
+
+
+    <main id="about-edit" class="container">
+      <div class="row section-banner">
+        <div class="col-md-12 section-header">
+          <h1>Edit Profile</h1>
+        </div>
+      </div>
+      <form>
+        <div class="row">
+          <div class="col-md-4 col-md-offset-1">
+            <div class="row">
+              <div class="col-md-12">
+                <h3>Basic Information</h3>
+              </div>
+            </div>
+            <div class="row extra-line-height">
+              <div class="col-xs-4">
+                <strong>Birthday:</strong>
+              </div>
+              <div class="col-xs-8">
+                July 31, 1980
+              </div>
+            </div>
+            <div class="row extra-line-height">
+              <div class="col-xs-4">
+                <strong>College:</strong>
+              </div>
+              <div class="col-xs-8">
+                <input type="text" value="Hogwart's College">
+              </div>
+            </div>
+            <div class="row extra-line-height">
+              <div class="col-xs-4">
+                <strong>Hometown:</strong>
+              </div>
+              <div class="col-xs-8">
+                <input type="text" value="Godrick's Hollow">
+              </div>
+            </div>
+            <div class="row extra-line-height">
+              <div class="col-xs-4">
+                <strong>Currently lives:</strong>
+              </div>
+              <div class="col-xs-8">
+                <input type="text" value="Godrick's Hollow">
+              </div>
+            </div>
+            <div class="row">
+              <div class="col-xs-12">
+                <h3>Contact Information</h3>
+              </div>
+            </div>
+            <div class="row extra-line-height">
+              <div class="col-xs-4">
+                <strong>Email:</strong>
+              </div>
+              <div class="col-xs-8">
+                harry_potter@hogwarts.edu
+              </div>
+            </div>
+            <div class="row extra-line-height">
+              <div class="col-xs-4">
+                <strong>Telephone:</strong>
+              </div>
+              <div class="col-xs-8">
+                <input type="tel" value="555-123-4567">
+              </div>
+            </div>
+          </div>
+          <div class="col-md-5 col-md-offset-1">
```

Comment:
Columns might not add up to 12 (col-md-4 col-md-offset-1 and col-md-5 col-md-offset-1). I'm not sure how much of an issue that is, but it could affectm mobile layouts.